scss/_nav.scss

```diff
@@ -158,6 +159,12 @@
     background: none;
     border: 0;
     @include border-radius(var(--#{$prefix}nav-pills-border-radius));
+
+    &:disabled {
```

Comment:
Is this really needed ? (Not sure about this one, just a ping) / do we need to add `.disabled`?